src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -116,6 +140,36 @@ const Authenticate = (() => {
         enableDisableSubmit();
     };
 
+    const onFileSelectedUns = (event) => {
```

Comment:
does `onFileSelectedUns` mean `onFileSelectedUnsupported`? If yes, it's okay to have long names if necessary, can you please rename that so it's clearer?

Edit: I see that it's used a lot in the file. I guess we can keep it. Perhaps you can leave one line of comment after the `require()` lines to let people know that it's shortened to `Uns`?